src/main/constants/contextKeys.ts

```diff
@@ -3,4 +3,17 @@ export class ContextKeys {
     public static SHOW_IN_SOURCE_CODE_ENABLED: string = 'jfrog.source.code.scan.jumpToSource.enabled';
     public static EXCLUDE_DEPENDENCY_ENABLED: string = 'jfrog.xray.excludeDependency.enabled';
     public static UPDATE_DEPENDENCY_ENABLED: string = 'jfrog.xray.updateDependency.enabled';
+    public static SET_SESSION_STATUS_KEY: string = 'jfrog.connection.status';
```

Comment:
It is obvious from the name of the enum that all of the enum context are keys
```suggestion
    public static SESSION_STATUS: string = 'jfrog.connection.status';
```